dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -26,6 +27,14 @@ const CMD_OMDB: &str = env!("CARGO_BIN_EXE_omdb");
 type ControlPlaneTestContext =
     nexus_test_utils::ControlPlaneTestContext<omicron_nexus::Server>;
 
+fn assert_oximeter_list_producers_output(output: &str, ox_url: &str) {
+    assert!(
+        output.contains(format!("Collector ID: {}", OXIMETER_UUID).as_str())
+    );
+    assert!(output.contains(PRODUCER_UUID));
+    assert!(output.contains(ox_url));
```

Comment:
This is the server address vs the specific producer. But, I can make this stronger.

Reply:
output is:

```
output: EXECUTING COMMAND: omdb ["oximeter", "list-producers"]
termination: Exited(0)
---------------------------------------------
stdout:
Collector ID: 39e6175b-4df2-4730-b11d-cbc1e60a2e78

Last refresh: Never

ID                                   ADDRESS         INTERVAL
a6458b7d-87c3-4483-be96-854d814c20de [::1]:37524     1s
e8a4203d-a6f1-4799-b982-f1b6235441db 127.0.0.1:38091 10s
---------------------------------------------
stderr:
note: using Oximeter URL http://[::1]:35380/
=============================================
```

Reply:
Oh, right, my mistake! I thought we were passing in the producer server, since much of our conversation from last week was about the variability in that.